packages/pieces/community/matrix/package.json

```diff
@@ -1,4 +1,7 @@
 {
   "name": "@activepieces/piece-matrix",
-  "version": "0.3.4"
-}
\ No newline at end of file
+  "version": "0.3.4",
+  "dependencies": {
+    "marked": "^15.0.6"
```

Comment:
Can you please run `npm install marked` in the global `package.json` and remove it from the dependencies section?  

We use Nx (nx.dev), which automatically detects dependencies during build time and includes them in the generated `package.json`. This approach helps us maintain a single version of each dependency in our `package.json`.

Reply:
ok should be fixed